Helpers/PaymentPlansHelper.php

```diff
@@ -153,9 +159,16 @@ public function planLabelByKey(string $key): string
         if (isset($matches[1])) {
             $label =  __('Pay in %1 installments', $matches[1]);
         }
+        // TODO refactor with basket items PR
+        if (isset($matches[1]) && $matches[1] === '1'  & $matches[2] == 0 && $matches[3] == 0) {
```

Comment:
matches[1] is in `string` ? and matches[2] and [3] in `int` ?

Reply:
`===` make a verification on the value and the type. `==` make a verification only on the value.